source/install/cloud-changelog.md

```diff
@@ -4,12 +4,61 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2023-06-29](#release-2023-06-29)
 - [Release 2023-06-13](#release-2023-06-13)
 - [Release 2023-05-31](#release-2023-05-31)
 - [Release 2023-05-01](#release-2023-05-01)
 - [Release 2023-04-21](#release-2023-04-21)
 - [Release 2023-03-29](#release-2023-03-29)
-- [Release 2023-03-20](#release-2023-03-20)
+
+## Release 2023-06-29
+
+### Highlights
+ - Insights has been disabled for all new instances and for existing servers that upgrade to v8.0. See more details in [this forum post](https://forum.mattermost.com/t/proposal-to-revise-our-insights-feature-due-to-known-performance-issues/16212) on why Insights has been disabled.
+ - Boards is now operating as a plugin and can be enabled or disabled in the **System Console > Plugin settings**. The Boards configuration, ``EnablePublicSharedBoards``, was also removed from the config and the **System Console**.
+ - The Channel Export and Apps plugins are now disabled by default.
+
+### Improvements
+
+#### User Interface (UI)
+ - Added support to specify different desktop notification sounds per channel.
+ - Calls: Ringing sounds can be enabled/disabled and selected in the Desktop Notifications preferences panel.
+ - Bumped the pre-packaged version of NPS to 1.3.2.
+
+#### Administration
+ - Playbooks is again shipped as a prepackaged plugin, with version 1.36.1.
+ - Added a new ``ConfigurationWillBeSaved`` plugin hook which is invoked before the configuration object is committed to the backing store.
+ - Admins can now specify index names to ignore while purging indexes from Elasticsearch with the ``ElasticsearchSettings.IgnoredPurgeIndexes`` setting.
+ - Added an option to use the German HPNS notification proxy.
+ - New flags were added to the database migrate command as following:
+
+    - ``auto-recover``: If the migration plan receives an error during migrations, this command will try to rollback migrations already applied within the plan. This option is not recommended to be added without reviewing migration plan. You can review the plan by combining ``--save-plan`` and ``--dry-run`` flags.
+    - ``save-plan``: The plan for the migration will be saved into the file store so that it can be used for reviewing the plan or to be used for downgrading.
+    - ``dry-run``: Does not apply the migrations, but it validates how the migration would run with the given conditions.
+
+ - A new database subcommand "downgrade" was added to be able to rollback database migrations. The command either requires an update plan to rollback, or comma separated version numbers.
```

Comment:
isn't this too abstract? should we point any documentation somewhere as an example?

Reply:
The docs update PR is here: https://github.com/mattermost/docs/pull/6491

When this PR is merged:

- the updated db migrate flags documentation will live here: ``manage/command-line-tools.html#mattermost-db-migrate``

- and the new subcommand docs page will be: ``manage/command-line-tools.html#mattermost-db-downgrade``